README.md

```diff
@@ -12,10 +12,15 @@ Documentation is available at https://docs.databricks.com/dev-tools/cli/databric
 
 This CLI is packaged as a dependency-free binary executable and may be located in any directory.
 See https://github.com/databricks/cli/releases for releases and
-[the docs pages](https://docs.databricks.com/dev-tools/cli/databricks-cli.html) for
-installation instructions.
+[the docs pages](https://docs.databricks.com/en/dev-tools/cli/install.html) for supported methods to install the CLI and the corresponding instructions.
 
 ------
+### Homebrew
+
+We maintain a [homebrew tap](https://github.com/databricks/homebrew-tap) for installing the Databricks CLI. You can find instructions for how to install, upgrade and downgrade the CLI using homebrew [here](https://docs.databricks.com/en/dev-tools/cli/install.html).
```

Comment:
This is the same link as above. Intentional?

Homebrew should be capitalized.

Reply:
Thanks, fixed.